ptx/sec_deriv_basic_rules.ptx

```diff
@@ -232,7 +232,17 @@
           <caption>A graph of <m>f(x) = x^3</m>, along with its derivative <m>\fp(x) = 3x^2</m> and its tangent line at <m>x=-1</m></caption>
           <!-- START figures/fig_xcubedwithderiv.tex -->
           <image xml:id="img_xcubedwithderiv" width="47%">
-            <description></description>
+            <shortdescription>
+              Graph of function x^3, its derivative and a tangent line drawn at point (-1,-1).
```

Comment:
Apparently I'm starting picky: I think you need an 'Oxford comma' after derivative.